shell/platform/tizen/tizen_renderer_ecore_wl2.cc

```diff
@@ -300,11 +300,17 @@ bool TizenRendererEcoreWl2::SetupEcoreWlWindow(int32_t width, int32_t height) {
   ecore_wl2_window_ =
       ecore_wl2_window_new(ecore_wl2_display_, nullptr, x, y, width, height);
   ecore_wl2_window_type_set(ecore_wl2_window_, ECORE_WL2_WINDOW_TYPE_TOPLEVEL);
-  ecore_wl2_window_alpha_set(ecore_wl2_window_, EINA_FALSE);
   ecore_wl2_window_position_set(ecore_wl2_window_, x, y);
   ecore_wl2_window_aux_hint_add(ecore_wl2_window_, 0,
                                 "wm.policy.win.user.geometry", "1");
 
+  // Makes the window transparent unless it's showing in full screen.
+  if (initial_geometry_.w > 0 || initial_geometry_.h > 0) {
```

Comment:
How about ``initial_geometry_.x``, ``initial_geometry_.y`` ?

Reply:
`w == 0 && h == 0` means the maximum window width and height by definition. `x` and `y` do not have such meaning and either could be 0 regardless of the window size.

Reply:
I made such a suggestion because it may not have any meaning if the width and height are set large. 
I thought that if x and y were apart from origin , there should be a transparent part unconditionally. As you said, even at the origin, if the width and height are small, it should be transparent.

I'm starting to think differently... How about these flags be set as a separate setting. 
Because even if the window is full size, you may want to set it to be transparent(Maybe.. video hole?) or vice versa.

Reply:
@xuelian-bai Can we make all windows transparent by default? What kind of problem there would be if we set alpha for full screen windows?

Reply:
I think we can't set them transparent by default,  from graphics point of view, if transparent is set, when rendering, e20 will have to render the window beneath the transparent window, that's not what we want. 
I agree with @bwikbs that, we may add another setting to set transparency. On TV, transparency is related with window size, but on other device, it may not.

Reply:
I created a property `transparent` in `FlutterDesktopWindowProperties`.